Android/app/src/main/java/app/intra/net/socks/SocksServer.java

```diff
@@ -42,6 +42,11 @@
 
   private final Context context;
 
+  private boolean useTlsWorkaround = false;
+  void enableTlsWorkaround(boolean v) {
```

Comment:
Could you add a comment briefly explaining the TLS workaround?

Reply:
Done